### ISSUE TITLE
Split window across multiple monitors

### ISSUE THREAD

**Issue by genemerewether** [rerun-io/rerun#6034]
Not sure if Rerun's windowing system supports this, but could we split the app into multiple windows (for use with multi-monitor / multi-display)?

**Comment by emilk** [comment]
`eframe` supports it yes, but we haven't implemented it for Rerun yet. It needs some design to figure out what it would mean (e.g. would be have multiple selection panels? Or have one main Rerun window, and then just pull out separate space views to their own windows?)

**Comment by alik-git** [comment]
what if I just want to have two different rerun viewers open? right now if a rerun viewer is already open, and I type "rerun" in my terminal, it connects to the already open viewer. but what if I just want another completely separate one running? is there a command for that?

**Comment by Wumpf** [comment]
> is there a command for that?

yes, you have to choose a different port than the default port to listen to. `rerun`  listens to `9876` by default and if it sees a rerun process already listening to this port it can't open a new viewer. You can explicitly set the port via `rerun --port 1234`

**Comment by jasperday-flowcopter** [comment]
Jumping in on this issue, this would be very useful for us (visualizing many simultaneous data streams, generally across three or four monitors). Keeping a single main Rerun window and being able to pull out the space views / containers into independent windows would be a massive benefit. If I have some extra bandwidth over the next month I could try to PR this.

**Comment by jasperday-flowcopter** [comment]
I see there's been a fair bit of conversation and PoCs already around this; I'll collect a little bit of it here:

- https://github.com/rerun-io/egui_tiles/pull/31
- https://github.com/rerun-io/egui_tiles/issues/30
- https://github.com/Latias94/egui_docking -- very good looking drag & drop docking setup, but probably not directly compatible with egui_tiles used by rerun
